Formula/global.rb

```diff
@@ -4,6 +4,7 @@ class Global < Formula
   url "https://ftpmirror.gnu.org/global/global-6.5.6.tar.gz"
   mirror "https://ftp.gnu.org/gnu/global/global-6.5.6.tar.gz"
   sha256 "122f9afa69a8daa0f64c12db7f02981fe573f51a163fa3829ed4f832cd281505"
+  revision 1
```

Comment:
It definitely doesn't so you can remove it, thanks. A `revision` will force a reinstall and, in some cases, rebuild for everyone using this.